test/e2e_node/podresources_test.go

```diff
@@ -739,3 +737,7 @@ func enablePodResourcesFeatureGateInKubelet(f *framework.Framework) (oldCfg *kub
 
 	return oldCfg
 }
+
+func isIntegral(cpuRequest float32) bool {
+	return cpuRequest == float32(int(cpuRequest))
```

Comment:
I'm always wary when I see equality check with floats, been burned more times than I'm comfortable recalling.
If we keep the `cpuRequest` as `int`, the check should look like
```
func isIntegral(cpuRequest int) bool {
	return (cpuRequest % 1000) == 0
}
```
which is always safe and easy to verify.

Reply:
Definitely!